lmfdb/modular_forms/elliptic_modular_forms/views/templates/experimental/emf_new.html

```diff
@@ -198,7 +198,7 @@ <h2>Further Properties</h2>
           <th>Eigenvalue
         </tr>
       </thead>
-      <tbody>
+      <tbody
```

Comment:
I think an extra closing `>` was deleted here.

Reply:
Fixed it